Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -649,6 +649,11 @@ private MessageRangeCheckResult MessageRangeCheck(ICommonSession session, ICChat
             case ChatTransmitRange.NoGhosts:
                 initialResult = (data.Observer && !_adminManager.IsAdmin(session)) ? MessageRangeCheckResult.Disallowed : MessageRangeCheckResult.Full;
                 break;
+            // Frontier - prevent TVs from spamming the poor, poor admins
+            case ChatTransmitRange.GhostRangeLimitNoAdminCheck:
+                initialResult = (data.Observer && data.Range < 0) ? MessageRangeCheckResult.HideChat : MessageRangeCheckResult.Full;
+                break;
```

Comment:
The crucial difference here vs. GhostRangeLimit is the lack of an `!_adminManager.IsAdmin(session)` check.

Behaviour that seems reasonable for television consoles/surveillance monitors:
* Both ghosts and living crew should see a pop-up in-game when nearby.
* Living crew should have a chat entry when nearby a television/monitor to follow/recall detailed conversations, especially for a surveillance monitor.
* Ghosts (admin or not) should not have a chat log entry when out of range of a television/monitor.

There are no built-in ChatTransmitRange settings that meet all three criteria.

* `GhostRangeLimit` (upstream behaviour) doesn't meet the third point (admins receive duplicated messages).
* `NoGhosts` doesn't meet the first point (no ghost pop-up) or third points (admins receive duplicated messages).
* `HideChat` doesn't meet the second point (no log for the living).  **If this solution as-is is undesirable, this may be the simplest fix, but does cause changes to behaviour.**
* `Normal` doesn't meet the third point (all ghosts receive duplicated messages).

At the time of writing, I can't see another way how this can be nicely done.